### DISCUSSION TITLE
Bun v1.0.19

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#7785] [Announcements]
To install Bun v1.0.19:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade to Bun v1.0.19:
```bash
bun upgrade
```

**[Read Bun v1.0.19's release notes on Bun's blog](https://bun.sh/blog/bun-v1.0.19)**

### Thanks to 18 contributors for making Bun v1.0.19 possible

- [@amartin96](https://github.com/amartin96)
- [@bjon](https://github.com/bjon)
- [@cirospaciari](https://github.com/cirospaciari)
- [@colinhacks](https://github.com/colinhacks)
- [@dotspencer](https://github.com/dotspencer)
- [@dylan-conway](https://github.com/dylan-conway)
- [@Electroid](https://github.com/Electroid)
- [@Jarred-Sumner](https://github.com/Jarred-Sumner)
- [@mathiasrw](https://github.com/mathiasrw)
- [@mitchellh](https://github.com/mitchellh) (for libxev timer heap code)
- [@o-az](https://github.com/o-az)
- [@Osmose](https://github.com/Osmose)
- [@paperdave](https://github.com/paperdave)
- [@scotttrinh](https://github.com/scotttrinh)
- [@sirenkovladd](https://github.com/sirenkovladd)
- [@sirhypernova](https://github.com/sirhypernova)
- [@spicyzboss](https://github.com/spicyzboss)
- [@vlechemin](https://github.com/vlechemin)

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v1.0.18...bun-v1.0.19

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.0.19'>Bun v1.0.19</a>.</em>

**Comment by vivekrp** [comment]
I have removed bun-types and installed @types/bun using bun add -d @types/bun. Should I be changing the tsconfig.json from :

```json
    "types": [
      "bun-types"
    ]
```

to this:

```json

    "types": [
      "@types/bun"
    ]
```
    
 or it works out of the box without this in tsconfig?

**Reply by samhuk** [reply]
By default, Typescript will pick up all types from packages in the node_modules/@types directory.

However, manually defining `types` in tsconfig will stop that automatic behavior, and then you need to add them all manually
